packages/opentelemetry-tracing/src/export/BatchSpanProcessorBase.ts

```diff
@@ -77,6 +77,11 @@ export abstract class BatchSpanProcessorBase<T extends BufferConfig> implements
     if (this._isShutdown) {
       return;
     }
+
+    if (span.spanContext().traceFlags !== TraceFlags.SAMPLED) {
```

Comment:
```suggestion
    if (span.spanContext().traceFlags & TraceFlags.SAMPLED) {
```

`traceFlags` is actually a bitmask.